pkg/cloud/alibaba/assets/cloud-controller-manager-clusterrole.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cloud-controller-manager
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - persistentvolumes
+      - services
+      - secrets
```

Comment:
We recently removed the need for other CCMs to use a clusterrole with overly permissive secrets, is this definitely required by the alibaba CCM or could this use a role scoped to the required namespaces?

Reply:
i'll investigate this